src/lib/actor.js

```diff
@@ -0,0 +1,19 @@
+export default class Actor{
+    constructor(ctx, stage){
+        this.ctx = ctx;
+        this.stage = stage;
+    }
```

Comment:
We can probably actually remove the ctx parameter, since stage objects will have a ctx field. 
Also add an options parameter, which accepts a bounds property, which contains the dimensions of the actor:
```
bounds = {
    x:0, 
    y:0,
    width: 0,
    height:0
}
```
```suggestion
constructor(stage, options) {
this.stage = stage;
this.ctx = stage.ctx;
this.bounds = options.bounds | {x: 0, y: 0, width:0, height: 0};
}
```